docs/rules/jsx-uses-react.md

```diff
@@ -39,6 +39,19 @@ var Foo = require('foo');
 var Hello = <div>Hello {this.props.name}</div>;
 ```
 
+As an alternative to specifying the above pragma in each source file, you can include an
+additional option in your rule configuration.
+
+```js
+[2, 'Foo']
+```
+
+```js
+var Foo = require('Foo');
+
+var Hello = <div>Hello {this.props.name}</div>;
+```
+
```

Comment:
Great!

Just some little things: can you make a Rule Options section and use the same format as the other rules for the config file example ?

You can see an example here: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-prop-types.md#rule-options